### PR TITLE
fixes to 12.40, 12.30, 12.20, 12.00 items and item movements

### PR DESCRIPTION
**Pull request by thatmichaelguy**
credit to @aurelion5670 for the updates

### CODE REVIEW

File: data/items/items.xml [hunk 1]
```diff
@@ -44981,7 +44991,7 @@
 		<attribute key="quicklootcategory" value="valuable" />
 		<attribute key="weight" value="150" />
 	</item>
-	<item id="34699" article="a" name="giant amethyst">
+	<item id="34699" article="a" name="giant sapphire" >
```
Comment (majestyotbr) [inline data/items/items.xml:44994]: whitespace

File: data/items/items.xml [hunk 2]
```diff
@@ -49301,8 +49295,10 @@
 	<item id="37539" article="a" name="phantasmal hair" >
 		<attribute key="weight" value="50" />
 	</item>
-	<item id="37540" article="a" name="unknow" >
-		<attribute key="weight" value="4" />
+	<item id="37540" article="a" name="pair of old bracers" >
```
Comment (majestyotbr) [inline data/items/items.xml:49298]: whitespace

**Comment by thatmichaelguy** [comment]
@MajestyOTBR I removed all whitespace from the end of regular tags.
There is a mix of whitespace and no whitespace for self-closing tags (" /> OR "/>), let me know which you want and I will include it in this PR.

**Comment by majestyotbr** [comment]
@thatmichaelguy i think it's ok now. Thanks.

**Review by aurelion5670 — APPROVED** [review]

**Comment by gabz182** [comment]
Need updt monster loot.
[Warning - Monsters::loadMonster] Unknown loot item "bloody tears".
[Warning - Monsters::loadMonster] Cant load loot. data/monster/quests/feaster_of

**Review by majestyotbr — APPROVED** [review]

**Pull request merged**